src/containers/ConversionForm/index.tsx

```diff
@@ -205,6 +212,14 @@ const ConversionForm = ({
                     >
                         <Button type="default">Select file</Button>
                     </Upload>
+                    {fileToConvert && (
+                        <div
+                            className={styles.removeFileIcon}
+                            onClick={handleRemoveFile}
+                        >
+                            {CancelIcon}
+                        </div>
```

Comment:
Clickable `div` elements can cause accessibility issues since they don't always play nice with screen readers. Consider changing this to a button!

Reply:
Good call, made the selector more specific and added a couple style rules to get the button right.